app/views/events/index.html.erb

```diff
@@ -1,10 +1,12 @@
-<h1 class="mb-6">Events lists</h1>
-<div><%= link_to "Create new event", new_event_path %></div>
+<div class="p-10">
+  <h1 class="mb-6">Events lists</h1>
+  <div><%= link_to "Create new event", new_event_path,class: 'btn_4'  %></div>
 
-<% @events.each_with_index do |event, index| %>    
-  <h2><%= "Event no. #{format("%02d", (index + 1).to_s)} - #{event.title.capitalize}" %></h2>
-  <%= preview_event_image event if event.image_is_saved_and_exists? %>
-  <div><%= link_to 'Show event details', event_path(event) %></div>
-<% end %>
+  <% @events.each_with_index do |event, index| %>    
+    <h2><%= "Event no. #{format("%02d", (index + 1).to_s)} - #{event.title.capitalize}" %></h2>
+    <%= preview_event_image event if event.image_is_saved_and_exists? %>
+    <div><%= link_to 'Show details', event_path(event), class: 'btn-2 no-underline' %></div>
+  <% end %>
 
-<div class="mt-6"><%== pagy_nav(@pagy) if @pagy.pages > 1 %></div>
+  <div class="mt-6"><%== pagy_nav(@pagy) if @pagy.pages > 1 %></div>
+</div>
```

Comment:
Same here.
Add an empty line at the end of your code

Reply:
fixed!!